src/Bootstrapper/Navigation.php

```diff
@@ -82,8 +82,13 @@ class Navigation extends RenderedObject
      * @var bool Whether the navigation links float right or not
      */
     protected $right = false;
+	
+	/**
+     * @var bool Whether the navigation links float left or not
+     */
+	protected $left = false;
 
```

Comment:
We use PSR-2 for our code style - can you change this to using 4 spaces instead of tabs?